helpers/postgresql.py

```diff
@@ -253,18 +275,37 @@ def start(self):
         ret = subprocess.call(self._pg_ctl + ['start', '-o', self.server_options()]) == 0
         ret and self.load_replication_slots()
         self.save_configuration_files()
-        if self.on_change_callback:
-            self.on_change_callback('replica' if os.path.exists(self.recovery_conf) else 'master')
+        if ret and 'on_start' in self.callback:
```

Comment:
We can, although it's the question of whether ON_START is less magical than "on_start" :-)